scripts/genproto.sh

```diff
@@ -76,6 +76,8 @@ protoc -I. \
     --swagger_out=logtostderr=true:./Documentation/dev-guide/apispec/swagger/. \
     ./etcdserver/etcdserverpb/rpc.proto
 
+rm -rf ${GOPATH}
```

Comment:
just in case something meddles with the gopath in between, do `rm -rf ${PWD}/gopath.proto` here?